forms-flow-bpm/forms-flow-bpm-camunda/src/main/java/org/camunda/bpm/extension/hooks/adapter/mail/CustomEmailConnector.java

```diff
@@ -0,0 +1,76 @@
+package org.camunda.bpm.extension.hooks.adapter.mail;
+
+import org.camunda.bpm.extension.hooks.adapter.mail.request.ConnectorRequest;
+import org.camunda.bpm.extension.hooks.adapter.mail.response.EmptyResponse;
+import org.camunda.bpm.extension.mail.MailConnectorException;
+import org.camunda.connect.impl.AbstractConnector;
+import org.camunda.connect.spi.ConnectorResponse;
+import org.formsflow.ai.bpm.mail.model.dto.FormsFlowBPMEmailRequestDto;
+import org.formsflow.ai.bpm.mail.service.FormsFlowBPMEmailAdapterService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+import java.util.logging.Logger;
+
+
+public class CustomEmailConnector extends AbstractConnector<ConnectorRequest, EmptyResponse> {
+
+    private final Logger LOGGER = Logger.getLogger(CustomEmailConnector.class.getName());
+
+    public static final String CONNECTOR_ID = "camunda-email-connector";
```

Comment:
Can we make the connector id generic ?

Reply:
This connector id is specific to the mail connector. I think we can make this as a common variable constants